components/Spinner/Spinner.jsx

```diff
@@ -1,4 +1,4 @@
 import React from 'react';
 import CircularProgress from 'material-ui/CircularProgress';
 
-export default ({ size = 1 }) => <div><CircularProgress size={size} /></div>;
+export default ({ size = 1, color = '#00BCD4' }) => <div><CircularProgress size={size} color={color} /></div>;
```

Comment:
i'd say that this color might need to be in or from the palette if it's a commonly used theme color (if we change theme colors, do we need to change this color?). If it's not, it's not terribly important to put it in palette.